src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -39,7 +39,7 @@ const MainNavDroplist = Droplist.extend({
       lists: [{
         collection: this.collection,
         itemTemplate: hbs`
-          <span class="app-nav__picklist-icon">{{fa iconType icon}}</span>
+          <span class="app-nav__picklist-icon">{{fa icon.type icon.name classes=icon.classes}}</span>
```

Comment:
It would be really nice if we could do something like `{{fa ...icon}}` here, to automatically fill the prefix/name. I suppose the args would have to be `prefix, iconName` instead of `type, name`.